bin/bump-version

```diff
@@ -14,18 +14,10 @@ while getopts "p:" o; do
   esac
 done
 
-echo "$patch_level"
-
 if [[ -z "${patch_level}" ]]; then
   usage
 fi
 
+# TODO catch if `npm version` returns non-zero exit code
```

Comment:
TODO before merge

Reply:
I fixed it in this PR rather than on `main` as here I only need to catch exit code from one command, vs on `main` there's `git` commands and a `gh` command that also need catching...

Simpler to just go with this route.